superset-frontend/src/SqlLab/App.jsx

```diff
@@ -105,17 +105,14 @@ const sqlLabPersistStateConfig = {
           ...initialState.sqlLab,
         },
       };
-      // Filter out any user data that may have been persisted in an older version.
-      // Get user from bootstrap data instead, every time
-      result.sqlLab.user = initialState.sqlLab.user;
       return result;
     },
   },
 };
 
 export const store = setupStore({
   initialState,
-  rootReducers: reducers,
+  rootReducers: { ...reducers, user: userReducer },
```

Comment:
Could you add the user reducer to `src/SqlLab/reducers/index.js` instead?

Reply:
First I intended to do so but `SqlLab/reducers/index.js` will be consolidated to SPA store which is already has the userReducer (as well as it will make a circular ref issue)
I just add like this since this will be gone after SPA work.